### PR TITLE
Try CI fix

### PR DESCRIPTION
**Pull request by Warhedgehog**
This fixes temporarly the CI problems. 

Because of the formatting it seems like it changed everything but I've highlighted the parts I've actually changed

### CODE REVIEW

File: .github/workflows/ci.yaml
Comment (Warhedgehog) [inline .github/workflows/ci.yaml:146]: This was changed

File: be1-go/Makefile
Comment (Warhedgehog) [inline be1-go/Makefile:9]: This was changed

File: .github/workflows/ci.yaml
Comment (Warhedgehog) [inline .github/workflows/ci.yaml:112]: This was changed

File: .github/workflows/ci.yaml
Comment (Warhedgehog) [inline .github/workflows/ci.yaml:149]: This was added

**Comment by lhmerino** [comment]
Looks good to me and I'll approve the pull request; if you need someone with knowledge about the CI, please wait for someone else to approve the pull request before merging.

**Review by lhmerino — APPROVED** [review]

**Review by Robingoumaz — APPROVED** [review]

**Review by nkcr — APPROVED** [review]

**Pull request merged**
